dev/com.ibm.ws.install.packaging_fat/fat/src/com/ibm/ws/install/packaging/fat/CheckTags.java

```diff
@@ -134,10 +134,10 @@ public void testSwidtagsExist() throws Exception {
             unexpected_file_found = true;
         }
 
-        testsPassed = ol_swidtag_exists && IBM_ol_swidtag_exists && !unexpected_file_found;
+        testsPassed = !ol_swidtag_exists && IBM_ol_swidtag_exists && !unexpected_file_found;
```

Comment:
Are we keeping `tag1` to make sure it doesn't exist? Then I think we should update the comments on line 106 and 119.